bundles/org.openhab.binding.bticinosmarther/src/main/java/org/openhab/binding/bticinosmarther/internal/api/dto/Chronothermostat.java

```diff
@@ -150,8 +151,12 @@ public String getActivationTime() {
         String timeLabel = TIME_FOREVER;
         if (activationTime != null) {
             try {
-                final ZonedDateTime dateActivationTime = DateUtil.parseZonedTime(activationTime, DTF_DATETIME_EXT);
-                final ZonedDateTime dateTomorrow = DateUtil.getZonedStartOfDay(1, dateActivationTime.getZone());
+                boolean dateActivationTimeIsZoned = activationTime.length() > 19;
+
+                final ZonedDateTime dateActivationTime = DateUtil.parseZonedTime(activationTime,
+                        dateActivationTimeIsZoned ? DTF_DATETIME_EXT : DTF_DATETIME);
+                final ZonedDateTime dateTomorrow = DateUtil.getZonedStartOfDay(1,
+                        dateActivationTimeIsZoned ? dateActivationTime.getZone() : TimeZone.getDefault().toZoneId());
```

Comment:
Which time-zone is expected here? Server time-zone or the time-zone configured in openHAB? In case of the latter, we would need the time-zone from [TimeZoneProvider](https://www.openhab.org/javadoc/latest/org/openhab/core/i18n/timezoneprovider).

Reply:
Ah thanks, i wasn't aware of the TimeZoneProvider, but that seems a better fit.